docs/tools-reference.md

```
@@ -69,6 +70,7 @@ Complete reference for all MCP tools provided by the Play Store MCP server.
 | Tool | Description |
 |---|---|
 | [`get_order`](#get_order) | Get order/transaction details |
+| [`batch_get_orders`](#batch_get_orders) | Get details for multiple orders at once |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)